### ISSUE TITLE
Error: Cannot find module occurred

### ISSUE THREAD

**Issue by bokuweb** [Quramy/reg-suit#19]
If known or unnecessary, please close this issue 🙏 

**version**: v0.017

`Error: Cannot find module` occurred when I try to initialize `reg-notify-github-plugin` with `init` command.

```
{ Error: Cannot find module 'reg-keygen-git-hash-plugin/lib/commit-explorer'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/username/sample/node_modules/reg-notify-github-plugin/lib/github-notifier-plugin.js:3:27)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3) code: 'MODULE_NOT_FOUND' }
```

**Comment by bokuweb** [comment]
Succeeded after installing `reg-keygen-git-hash-plugin`.

**Comment by Quramy** [comment]
I've fixed it at the above commit and I've released the version 0.0.18.

Thanks for your reporting :)
